v1/test/e2e/lib/util.sh

```
@@ -45,6 +45,27 @@ function cleanup_conn() {
     [[ -n $PID_CONN ]] && kill $PID_CONN
 }
 
+function download_binary() {
+    url=$1
+    name=$2
+
+    max_attempts=3
+    attempt=1
+    while [[ $attempt -le ${max_attempts} ]]; do
+        if curl -fL $url -o $name; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)